common/logger/src/libra_logger.rs

```diff
@@ -250,19 +290,14 @@ impl Writer for StderrWriter {
 /// UNIX_TIMESTAMP LOG_LEVEL FILE:LINE MESSAGE
 /// Example:
 /// 2020-03-07 05:03:03 INFO common/libra-logger/src/lib.rs:261 Hello
-fn format(entry: &StructuredLogEntry) -> Result<String, fmt::Error> {
+fn format(entry: &LogEntry) -> Result<String, fmt::Error> {
     use std::fmt::Write;
 
     let mut w = String::new();
-    if let Some(level) = &entry.level {
-        write!(w, "{} ", level)?;
-    }
-
+    write!(w, "{} ", entry.metadata.level())?;
     write!(w, "{} ", entry.timestamp)?;
 
-    if let Some(location) = &entry.location {
-        write!(w, "{} ", location)?;
-    }
+    write!(w, "{}:{} ", entry.metadata.file(), entry.metadata.line())?;
```

Comment:
Any reason you don't write these all in one format string?

Also, I see there some advantages to splitting the file and the line, aka being able to search by file.  However, being able to search by line is less helpful.  I think there is the disadvantage that, here we format it all into the familiar `file:line` format, but we'd have to do extra wrangling to see that information easily in Kibana.

We can still prefix search for the file name in Kibana, so I'm not sure if this split is necessary.

Reply:
The big reason was that the old StrucuturedLogEntry didn't have these fields required so you had  to unwrap them to print them. Now theres not that big of a reason to.

From a structure pov, we should keep them separate, as they are different pieces of data and i don't know if its really that much more difficult to look at a `line` field to find the line number. If you feel strongly enough though, we can add it back